community/dosbox/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+./autogen.sh
```

Comment:
Does this run `autoconf` and friends? If so they should be added to `depends`. :+1:

Reply:
Yes, I did consider that.  So to test I did "kiss r autoconf", and then "kiss b dosbox" again, and it still worked.  So I guess it doesn't need autoconf?

Reply:
Hmm, the script just calls autoconf and friends anyway, so I think it should probably be in the deps list.  Will add.

Reply:
Does the script need to be run? If the build works without it, it may just skip calling them if unneeded.

Reply:
It seems that it can work on my system without running autogen.sh first, though perhaps that may not be true for other people? I guess what is guidestone on this... "If autogen.sh is present, don't run it if you don't need to"?

Reply:
Yeah. Will add it to the list tomorrow. We avoid running the `auto*` stuff if unneeded.

Reply:
Got it - will  remove running of autogen.sh from build and remove autoconf from deps.